Applications/MaterialManager/Samples/Create/Boards/Readme.md

```diff
@@ -0,0 +1,27 @@
+<h1 id="createBoardType"> Create board type</h1>
+
+With the Homag Connect materialManager boards client, new board types can be created.
```

Comment:
Please double check the overall wording (also with other apps docu).

e.g. Homag -> HOMAG

Reply:
I checked the wording and made sure that it is HOMAG everywhere